### ISSUE TITLE
Provide feasible start solution

### ISSUE THREAD

**Issue by dirkschumacher** [dirkschumacher/ompr#59]
A user should be able to provide a start solution

**Comment by HadhadMohamed** [comment]
Hi Dirk,
How can i inject a start solution on my program

**Comment by dirkschumacher** [comment]
You cannot at the moment.

**Comment by HadhadMohamed** [comment]
You said that a user should be able to provide a startup solution. Indeed I have an initial solution how to call it in my script or how to use it to initialize the program.

**Comment by dirkschumacher** [comment]
Yes, that would be nice, but it is currently not implemented. I am not even sure the specific solver packages in R implement this feature.

**Comment by rickyars** [comment]
I would love something like this.
